src/components/Card/Card.jsx

```diff
@@ -1,23 +1,113 @@
+                {output}
+            </div>
+        </>
     )
 }
 
-
 Card.propTypes = {
 	heading: PropTypes.string.isRequired,
 	text: PropTypes.string,
     list: PropTypes.array,
 }
+
+export function FormCard( {heading, sendDataOnSubmit} ) {
+    const [name, setName] = useState('');
+    const [email, setEmail] = useState('');
+    const [subject, setSubject] = useState('');
+    const [formText, setFormText] = useState('');
+
+    function onNameChange(event) {
+        setName(event.target.value);
+    }
+    function onEmailChange(event) {
+        setEmail(event.target.value);
+    }
+    function onSubjectChange(event) {
+        setSubject(event.target.value);
+    }
+    function onFormTextChange(event) {
+        setFormText(event.target.value);
+    }
+    function onSubmit(event) {
+        event.preventDefault();
+        sendDataOnSubmit({
+            name: name,
+            email: email,
+            subject: subject,
+            formText: formText,
+        });
+    }
+
+    return (
+        <div className={`p-3 mb-3 bg-white border border-secondary 
+            rounded-3 ${styles.formTile} ${styles.tile}`}>
+            <h2>{heading}</h2>
+            <form onSubmit={onSubmit}>
+                <FormInputWithLabel
+                    labelText="Name"
+                    inputType="text"
+                    inputId="inputName"
+                    inputValue={name}
+                    inputOnChange={onNameChange}/>
+                <FormInputWithLabel
+                    labelText="E-Mail-Adresse"
+                    inputType="email"
+                    inputId="inputEmail"
+                    inputValue={email}
+                    inputOnChange={onEmailChange}/>
+                <FormInputWithLabel
+                    labelText="Betreff"
+                    inputType="text"
+                    inputId="inputSubject"
+                    inputValue={subject}
+                    inputOnChange={onSubjectChange}/>
+                <FormInputWithLabel
+                    labelText="Text"
+                    inputType="textarea"
+                    inputId="inputText"
+                    inputValue={formText}
+                    inputOnChange={onFormTextChange}/>
+                <button type="submit"
+                        className="btn btn-primary">
+                    Do Magic!
+                </button>
+            </form>
+        </div>
+    )
+}
+
+export function DbCard( {heading, contacts} ) {
+
+    return (
+        <>
+            <div className={`p-3 mb-3 bg-white border border-secondary rounded-3
+                    animate__animated animate__backInRight ${styles.formTile} ${styles.tile}`}>
+                <h2>{heading}</h2>
+                <ul>
+                    {contacts.map((contact) => (
+                        <li>
+                            <strong>{contact.name}</strong> ({contact.email}): {contact.subject} - {contact.formText}
+                        </li>
```

Comment:
Hier wird der Browser vermutlich auch meckern wegen des fehlenden keys. Bei jedem mapping sollte auch ein key angegeben werden.
```suggestion
                    {contacts.map((contact, index) => (
                        <li key={index}>
                            <strong>{contact.name}</strong> ({contact.email}): {contact.subject} - {contact.formText}
                        </li>
```